components/FlashCardList.js

```diff
@@ -32,9 +40,12 @@ export default function FlashCardList({
             card={card}
             onMarkCorrect={onMarkCorrect}
             collections={collections}
+            deleteCard={deleteCard}
           />
         ))}
-        {flashCards.length === 0 && <p>{emptyListMessage}</p>}
+        {flashCards.length === 0 && (
+          <StyledEmptyListMessage>{emptyListMessage}</StyledEmptyListMessage>
+        )}
       </StyledList>
```

Comment:
The direct child elements of a `ul` should be `li` elements. Please move the `StyledEmptyListMessage` outside your list.